build.gradle

```diff
@@ -4,15 +4,15 @@ buildscript {
             'androidXTestCore':'1.0.0',
             'androidXTestRules':'1.1.0',
             'assertJ': '3.16.1',
-            'compose': '1.0.0-alpha07',
-            'composeNavigation': '1.0.0-alpha02',
+            'compose': '1.0.0-alpha08',
+            'composeNavigation': '1.0.0-alpha03',
             'detekt': '1.7.4',
             'espresso': '3.2.0',
             'gradle': '4.2.0-alpha15',
             'junit' : '4.12',
             'junitImplementation' : '1.1.1',
-            'kotlin': '1.4.0',
-            'kotlinCompilerVersion': '1.4.0',
+            'kotlin': '1.4.20',
```

Comment:
1.4.21 is out now

Reply:
Don't think the dependencies that compose is using have started using that so it might not work. I'll still give it a shot but don't expect it to work.